src/main/resources/com/google/api/codegen/metadatagen/java/grpc/build.gradle.snip

```diff
@@ -150,11 +150,23 @@
     }
   }
 
-  // Regenerates the gh-pages branch under tmp_gh-pages, which must be committed separately
-  task updateDocsWithCurrentVersion {
+  // Regenerates the gh-pages branch under tmp_gh-pages
+  task updateDocs {
     dependsOn 'createApiDocsRedirect'
   }
 
+  // Regenerates and push the gh-pages branch under tmp_gh-pages
+  task updateAndPushDocs {
+    dependsOn 'createApiDocsRedirect'
```

Comment:
Done (I have refactored this section by merging 'updateDocs` with 'createApiDocsRedirect').